lib/checks/aria/deprecatedrole.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "deprecatedrole",
+  "evaluate": "deprecatedrole-evaluate",
+  "metadata": {
+    "impact": "minor",
```

Comment:
Should we make this somehow return incomplete rather than a failure? Merging this will start failing for something that passed for a non-major release.

Reply:
That's why it's a `feat`. New failures can come up from minor releases.